### PR TITLE
Update Main.lua

### PR DESCRIPTION
**Pull request by maphex**
Change search slugs for Mythic Plus Dungeons for Season 2 to match Raider.io Slugs

### CODE REVIEW

**Comment by creazy231** [comment]
fixes #172

**Comment by oscar-wos** [comment]
surely a nobrainer

File: Main.lua [hunk 1]
```diff
@@ -461,10 +461,8 @@ function PGF.DoFilterSearchResults(results)
         -- Battle for Azeroth dungeons
         --                 normal        heroic        mythic        mythic+       normal2
         env.ad           = aID == 501 or aID == 500 or aID == 499 or aID == 502 or aID == 543  -- Atal'Dazar
-        env.tosl         = aID == 503 or aID == 505 or aID == 645 or aID == 504 or aID == 542  -- Temple of Sethraliss
-        env.tos          = env.tosl
-        env.tur          = aID == 506 or aID == 508 or aID == 644 or aID == 507 or aID == 541  -- The Underrot
-        env.undr         = env.tur
+        env.tos         = aID == 503 or aID == 505 or aID == 645 or aID == 504 or aID == 542  -- Temple of Sethraliss
```
Comment (robertlyall) [inline Main.lua:464]:
```suggestion
        env.tos          = aID == 503 or aID == 505 or aID == 645 or aID == 504 or aID == 542  -- Temple of Sethraliss
```

File: Main.lua [hunk 2]
```diff
@@ -507,12 +505,12 @@ function PGF.DoFilterSearchResults(results)
         env.aa          = aID == 1157 or aID == 1158 or aID == 1159 or aID == 1160 -- Algeth'ar Academy
         env.bh          = aID == 1161 or aID == 1162 or aID == 1163 or aID == 1164 -- Brackenhide Hollow
         env.hoi         = aID == 1165 or aID == 1166 or aID == 1167 or aID == 1168 -- Halls of Infusion
-        env.nt          = aID == 1169 or aID == 1170 or aID == 1171 or aID == 1172 -- Neltharus
+        env.nelt          = aID == 1169 or aID == 1170 or aID == 1171 or aID == 1172 -- Neltharus
```
Comment (robertlyall) [inline Main.lua:508]:
```suggestion
        env.nelt        = aID == 1169 or aID == 1170 or aID == 1171 or aID == 1172 -- Neltharus
```

**Comment by 0xbs** [comment]
Removing the existing keywords would break expressions which are currenlty working fine. Also rio is not the measure of all things and their keywords are not better or worse than mine. I tried to align with rio in advance in the past, however communication always stalled so I don't bother much anymore.

There is a function `PutRaiderIOAliases` where I add the rio aliases. The keywords should go there.

Changing the tooltip is okay as most players will remember rio's keywords better than mine. However your change is not quite correct as line 62 should be changed as well.

I will decline this PR and add my changes directly to the master. Still thank you for your contribution!

**Pull request closed without merging**